lib/Controller/PublicViewController.php

```diff
@@ -120,6 +120,7 @@ private function publicIndex(string $token,
 		$defaultSkipPopover = $this->config->getAppValue($this->appName, 'skipPopover', 'yes');
 		$defaultTimezone = $this->config->getAppValue($this->appName, 'timezone', 'automatic');
 		$defaultSlotDuration = $this->config->getAppValue($this->appName, 'slotDuration', '00:30:00');
+		$defaultSyncTimeout = $this->config->getAppValue($this->appName, 'syncTimeout', '00:02:00');
```

Comment:
This format is kinda ugly and prone to issues. The [ISO 8601 duration format](https://en.wikipedia.org/wiki/ISO_8601#Durations) is probably better here (we already use if for [the refresh rate of subscriptions](https://docs.nextcloud.com/server/21/admin_manual/groupware/calendar.html#refresh-rate) for instance). On the front-end [you can use momentjs to parse it](https://momentjs.com/docs/#/durations/).

Reply:
I'm using ISO 8601 in the tests and defaults now and parse the value with `moment.duration` - so in the end both formats will work.

I'm not entirely sure what more it will take to have a properly working setting. I guess I will open another PR to https://github.com/nextcloud/documentation/blob/master/admin_manual/groupware/calendar.rst to document the setting.

When loading the app i am currently still seeing:
```
Initial settings:
	- AppVersion: 2.1.3
	- EventLimit: true
	- FirstRun: true
	- ShowWeekNumbers: false
	- ShowTasks: true
	- ShowWeekends: true
	- SkipPopover: false
	- SlotDuration: 00:30:00
	- SyncTimeout: undefined
	- TalkEnabled: false
	- TasksEnabled: false
	- Timezone: automatic
```
So I guess i am still missing something to get a valid default.